doc/sphinx/create_algorithms_rst.py

```diff
@@ -1,7 +1,13 @@
 import json
+import os
+from pathlib import Path
 from textwrap import dedent
+from typing import Any
 
-with open("algorithms.json") as f, open("source/algorithms.rst", "w") as fw:
+root_dir = Path(__file__).parent
+
+
+with open(root_dir / "algorithms.json") as f, open(root_dir / "source/algorithms.rst", "w") as fw:
```

Comment:
@nvitucci FYI neat way to not depend on the folder the script gets executed from :)

Reply:
Glad it helps :)